docs/Installation-Script/UnityStationInstaller.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
+# Fonction de gestion des erreurs
+function handle_error {
+    echo "Une erreur s'est produite. Sortie du script."
+    exit 1
+}
+
+# Fonction pour obtenir la dernière version depuis l'API GitHub
+function get_latest_version {
+    local repo_url="https://api.github.com/repos/unitystation/stationhub/releases/latest"
+    local version=$(wget -qO - "$repo_url" | grep -oP '"tag_name": "\K(.*)(?=")')
+    echo "$version"
+}
+
+latest_version=$(get_latest_version)
+
+while true; do
+    echo "Choose your distribution :"
+    echo "1 - Ubuntu/Debian"
+    echo "2 - Arch-Linux"
+    echo "3 - Fedora"
+    echo "4 - OpenSUSE"
+    echo "5 - CentOS"
+    echo "6 - Gentoo"
+    read -p "Enter the number of your choice : " choice
+
+    case $choice in
+        1)
+            # Commandes pour Ubuntu
+            apt update
+            apt full-upgrade
+            apt install -y wget p7zip || handle_error
+            ;;
+
+        2)
+            # Commandes pour Arch Linux
+            pacman -Syu --noconfirm
+            pacman -S --noconfirm wget p7zip || handle_error
+            ;;
+
+        3)
+            # Commandes pour Fedora
+            dnf update -y
```

Comment:
You said above:

> as I said earlier, there is no -y
so the user is free to do Y/N if the suggested updates don't suit, it doesn't block the script.

However this DOES force a user to update their system, please do not do this.